content/docs/self-hosted-runners.md

```diff
@@ -1,82 +1,81 @@
+runners by default. However, there are many great reasons to use your own
+runners: to take advantage of GPUs; to orchestrate your team's shared computing
+resources, or to train in the cloud.
 
 ☝️ **Tip!** Check out the official documentation from
 [GitHub](https://help.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners)
-and [GitLab](https://docs.gitlab.com/runner/) to get started setting up your
+and [GitLab](https://docs.gitlab.com/runner) to get started setting up your
 self-hosted runner.
 
-## Allocating cloud resources with CML
+## Allocating Cloud Compute Resources with CML
 
-When a workflow requires computational resources (such as GPUs) CML can
+When a workflow requires computational resources (such as GPUs), CML can
 automatically allocate cloud instances using `cml-runner`. You can spin up
-instances on your AWS or Azure account (GCP support is forthcoming!).
+instances on your [AWS](#aws), [Azure](#azure), [GCP](#gcp), or
+[Kubernetes](#kubernetes) accounts.
 
 For example, the following workflow deploys a `t2.micro` instance on AWS EC2 and
 trains a model on the instance. After the job runs, the instance automatically
-shuts down. You might notice that this workflow is quite similar to the
-[basic use case](/doc/usage) highlighted in the beginning of the docs- that's
-because it is! What's new is that we've added `cml-runner`, plus a few
-environmental variables for passing your cloud service credentials to the
+shuts down.
+
+You might notice that this workflow is quite similar to the
+[basic use case](/doc/usage). The only addition is `cml-runner` and a few
+environment variables for passing your cloud service credentials to the
 workflow.
 
+Note that `cml-runner` will also automatically restart your jobs (whether from a
+[GitHub Actions 72-hour timeout](https://docs.github.com/en/actions/reference/usage-limits-billing-and-administration#usage-limits)
+or a
+[AWS EC2 spot instance interruption](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-interruptions.html)).
+
 ```yaml
-name: 'Train-in-the-cloud'
+name: Train-in-the-cloud
 on: [push]
-
 jobs:
   deploy-runner:
-    runs-on: [ubuntu-latest]
+    runs-on: ubuntu-latest
     steps:
       - uses: iterative/setup-cml@v1
       - uses: actions/checkout@v2
-      - name: 'Deploy runner on EC2'
-        shell: bash
+      - name: Deploy runner on EC2
         env:
           REPO_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         run: |
           cml-runner \
-          --cloud aws \
-          --cloud-region us-west \
-          --cloud-type=t2.micro \
-          --labels=cml-runner
-  model-training:
+              --cloud aws \
+              --cloud-region us-west \
+              --cloud-type t2.micro \
+              --labels cml-runner
```

Comment:
I would recommend `--option=value` instead of `--option value` because some values like SSH keys begin with an hyphen, but users can notice that easily, way more easily than us: https://github.com/iterative/cml/pull/493.

Independently of the format we choose, we should consistently apply the same style to all the documentation and examples.

Reply:
updated everywhere